app/src/main/java/me/hufman/androidautoidrive/carapp/carinfo/CarDetailedInfo.kt

```diff
@@ -184,11 +198,19 @@ class CarDetailedInfo(carCapabilities: Map<String, Any?>, cdsMetrics: CDSMetrics
 			speed, heading,
 			gforces, engineRpm
 	)
+	private val drivingPerformanceFields: List<Flow<String>> = listOf(
+			drivingMode, drivingGearLabel,
+			accelContact, steeringAngle,
+			speed, torque,
+			gforces, heading,
+			engineTemp, oilTemp
+	)
 	private val gpsFields: List<Flow<String>> = listOf(
 			countryLabel, heading,
 			cityLabel, altitudeLabel,
 			streetLabel, crossStreetLabel,
 			latitudeLabel, longitudeLabel,
+			speedGPS
```

Comment:
This will answer #699, which will be great, thanks!

Reply:
For information, I've had implemented this feature in my own build since end of march (see https://www.1erforum.de/threads/spotify-mit-bmw-connected-app-funktioniert-nicht.255788/page-9#post-3179485)

Reply:
Neat! I like seeing when the app shows up in other places :)